bccsp/pkcs11/impl.go

```diff
@@ -151,14 +175,28 @@ func (csp *impl) KeyImport(raw interface{}, opts bccsp.KeyImportOpts) (k bccsp.K
 // GetKey returns the key this CSP associates to
 // the Subject Key Identifier ski.
 func (csp *impl) GetKey(ski []byte) (bccsp.Key, error) {
-	pubKey, isPriv, err := csp.getECKey(ski)
-	if err == nil {
-		if isPriv {
-			return &ecdsaPrivateKey{ski, ecdsaPublicKey{ski, pubKey}}, nil
-		}
-		return &ecdsaPublicKey{ski, pubKey}, nil
+	privHandle, pubHandle, pubKey, err := csp.getECKey(ski)
+	if err != nil {
+		logger.Warnf("Key not found using PKCS11: %v", err)
+		return csp.BCCSP.GetKey(ski)
```

Comment:
I expect this is going to happen quite frequently.

Reply:
I introduced this after Mihir and I were trying to debug an issue but couldn't trace the failure as none of there errors were logged and Gari picked it up as part of his cherry-pick. Should probably be raised in severity into `debug`. I'll make the change in master and cherry-pick it.